src/Parser.js

```diff
@@ -47,7 +47,7 @@ var ParseError = require("./ParseError");
  */
 function Parser(input, settings) {
     // Make a new lexer
-    this.lexer = new Lexer(input);
+    this.gullet = new MacroExpander(input, settings && settings.macros);
```

Comment:
Either that, or call the property `macroExpander`, `me` or something like that. Preferences?